resources/views/auth/_login_cover.blade.php

```diff
@@ -0,0 +1,63 @@
+@extends(backpack_view('layouts.auth'))
```

Comment:
Awesome! We can probably find a better image than the one Tabler chose, with no sweatpants 😅 so that it feels more "admin panel-y" but that's not urgent.